tic-tac-toe/src/app/screens/Game/index.js

```diff
@@ -4,21 +4,26 @@ import { connect } from 'react-redux';
 
 import Board from '../Game/components/Board';
 import { lines } from '../../../constants';
+import getBookDetail from '../../../services/BookService';
 import { clickprueba } from '../../../Game/actions';
 
 class Game extends Component {
   state = {
     history: [{ squares: Array(9).fill(null) }]
   };
 
-  calculateWinner = squares => {
-    for (let i = 0; i < lines.length; i += 1) {
-      const [a, b, c] = lines[i];
+  calculateWinner = async squares => {
+    const test = await getBookDetail();
```

Comment:
The name getBookDetail is not declarative. Change this name. Change the test name to lines for example.
The lines are the same always, so you don't need to ask to api a lot of times. You can use componentDidMount method and get the lines to the API there.